### PR TITLE
feat: Compare Entries [EXT-3741]

### PR DESCRIPTION
**Pull request by andipaetzold**
With this app, you can compare the current entry with the same entry in another environment.

image
image

### CODE REVIEW

**Review by shikaan — COMMENTED** [review]
LGTM, one comment: I wonder if we should make the diff in the dialog larger (to cover the whole area of the dialog). The way it looks rn it seems a bit strange to me

File: examples/compare-entries/src/components/LocalhostWarning.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (andipaetzold) [reply examples/compare-entries/src/components/LocalhostWarning.tsx:1]: Sounds good

**Comment by andipaetzold** [comment]
> I wonder if we should make the diff in the dialog larger (to cover the whole area of the dialog). The way it looks rn it seems a bit strange to me

You mean that the dialog always covers the full screen size and shows more of the code if more space is available? Not sure whether this is possible. Maybe I can increase the surrounding lines of a change or add a min height to the dialog.

**Comment by andipaetzold** [comment]
image

@shikaan 
Looks a little stretched now, but I guess that's fine

**Review by shikaan — APPROVED** [review]

**Pull request merged**